aspnetcore/migration/webapi/sample/ProductsApp/Controllers/ProductsController.cs

```diff
@@ -11,9 +11,18 @@ public class ProductsController : ApiController
     {
         Product[] products = new Product[] 
         { 
-            new Product { Id = 1, Name = "Tomato Soup", Category = "Groceries", Price = 1 }, 
-            new Product { Id = 2, Name = "Yo-yo", Category = "Toys", Price = 3.75M }, 
-            new Product { Id = 3, Name = "Hammer", Category = "Hardware", Price = 16.99M } 
+            new Product
```

Comment:
This is a total side-note/:question:/digression .... engineering :heart: putting a comma on the last prop/object. I never used to do it my own code, but I wonder if I should be doing it all the time now ... my code and samps/examples. I don't know if I've even been consistent about it. Do we want to make a little rule about it right here, right now?